src/external_connections/next_gen_network.py

```diff
@@ -14,6 +14,7 @@
 from functools import partial
 from itertools import chain, islice
 import next_gen_io
+import json
 
```

Comment:
Please change the example usage on line 3 to include the new flowpath_data.geojson file instead and remove the previous geojson file from the repo.